### PR TITLE
Adding pycentroids recipe

### PR DESCRIPTION
**Pull request by kbeeperez**
implements: https://github.com/nsls-ii-forge/pycentroids-feedstock

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by kbeeperez** [comment]
Attn @mrakitin

**Review by mrakitin — COMMENTED** [review]
The directory name is misspelled - `pycdntroids`, that should be `pycentroids`. Could you please fix it, @kbeeperez?

File: recipes/pycdntroids/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{% set name = "pycentroids" %}
+{% set version = "0.2.0rc2" %}
+{% set sha256 = "642109bffe3c2733c06e897b91bc999ea0e960981fea8ba74ddce25e90faf5fe" %}
```
Comment (mrakitin) [inline recipes/pycdntroids/meta.yaml:3]: This hash can be moved below to the `source` block as it's done in other recipes.

File: recipes/pycdntroids/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  imports:
+    - pycentroids
+  commands:
+    - pytest --pyargs pycentroids.tests
+
+about:
+  home: https://github.com/NSLS-II/centroids
+  license: BSD-3-Clause
+  license_file: LICENSE/LICENSE
+  summary: need to add a summary
```
Comment (mrakitin) [inline recipes/pycdntroids/meta.yaml:48]:
```suggestion
  summary: Routines for performing single-photon counting and centroiding of CCD data
```

**Review by mrakitin — COMMENTED** [review]
Added the summary and a suggestion about sha256 var.

@conda-forge/staged-recipes, I agree to be a maintainer of this package.

**Comment by mrakitin** [comment]
Thanks, @ocefpaf!

**Pull request merged**